pkg/datapath/linux/node.go

```diff
@@ -1368,6 +1368,50 @@ func (n *linuxNodeHandler) NodeNeighborRefresh(ctx context.Context, nodeToRefres
 	}
 }
 
+// NodeCleanNeighbors cleans all neighbor entries for the direct routing device
+// and the encrypt interface. It should be used when the agent changes the
+// state from `n.enableNeighDiscovery = true` to
+// `n.enableNeighDiscovery = false`.
+func (n *linuxNodeHandler) NodeCleanNeighbors() {
```

Comment:
This is called during agent `init`. Does this need to be a blocking call?

Reply:
correct